lascli/parser/datasets.py

```diff
@@ -44,4 +117,13 @@ def create_datasets_parser(subparsers):
     delete_dataset_parser.add_argument('--delete-documents', action='store_true', default=False)
     delete_dataset_parser.set_defaults(cmd=delete_dataset)
 
+    upload_batch_to_dataset_parser = subparsers.add_parser('sync')
+    upload_batch_to_dataset_parser.add_argument('dataset_id')
+    upload_batch_to_dataset_parser.add_argument('documents', default=False)
```

Comment:
I would add a help section here and/or change the name of the argument to better describe expected input e.g. documents_json_path